src/components/DocumentActions.tsx

```
@@ -72,6 +74,56 @@ const DocumentActions: React.FC<DocumentActionsProps> = ({
+	const handleImagePreview = () => {
+		try {
+			const parsed = JSON.parse(documentStatus?.doc_data as string);
+			const subject = parsed?.credentialSubject;
+
+			if (subject && typeof subject === 'object') {
+				// 2️⃣ Collect every nested value that:
+				//     • is an object
+				//     • has a `mimetype` beginning with "image/"
+				//     • has a non-empty `content` field
+				const imageEntries = Object.values(subject).filter(
+					(entry: any) =>
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)